packages/graphback-cli/src/helpers/db.ts

```diff
@@ -6,7 +6,7 @@ import { DatabaseSchemaManager, GraphQLBackendCreator } from 'graphback';
 import { logError, logInfo } from '../utils'
 import { checkDirectory } from './common'
 
-const configPath = `${process.cwd()}/config.json`
+const configPath = `${process.cwd()}/graphback.json`
```

Comment:
[Idea] Can we have function that will append all `process.cwd()` to all paths in config. Then we can have interface for it and pass it to every module to use it. This will be the only one `extra field` that we will need to require from outside as it is path to config itself.